### ISSUE TITLE
Feedback button is not visible because misinterpreting timeSlotVisible attribute as seconds than minutes

### ISSUE THREAD

**Issue by sippsack** [dukecon/dukecon_pwa#85]
At JavaLand 2019 people complained again and again about a not visible Feedback button. We could not reproduce the problem yet. Maybe it is a timeout for the keycloak token and the login button still says the person is logged in.

**Comment by sippsack** [comment]
e. g. "Hi,,

ich wollte noch ein paar Bewertungen vergeben, aber der Button erscheint nicht mehr. Ein paar habe ich heute Mittag noch bewertet, aber nun geht es nicht mehr. 
Habt Ihr die Bewertungen wieder deaktiviert?"

**Comment by ahus1** [comment]
The JavaLand production configuration allows giving feedback for up to 2,8 hours (10080 seconds). After that time no feedback is possible. Please note: admins can always give feedback.

I cleaned up some code to use computed instead of methods to make it reactive to login status changes. Please test. Thanks!

**Comment by sippsack** [comment]
you are right, a misunderstanding between server and client. I didn't recognize it because I'm an admin. 

the server interprets it the following way:
timeSlotVisible: -1 # in minutes from start of the talk, -1 means at any time (testing only)

10080 minutes should have been 7 days

**Comment by ahus1** [comment]
I didn't look at the server. I changed it in the client. The client now also supports "-1" as "always on". Please test. Please review and test, deploy to production and maybe adjust any changed configurations in production (and then close this ticket)
